docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -2469,6 +2469,44 @@ definitions:
 
           If this is set it MUST be a valid required field as defined in the schema.
 
+      cleanup_policy:
+        type: string
+        x-extensible-enum:
+          - delete
+          - compact
+        default: 'delete'
+        description: |
+          Event type cleanup policy. There are two possible values:
+
+          - 'delete' cleanup policy will delete old events after retention time expires. Nakadi guarantees that each
+          event will be available for at least the retention time period. However Nakadi doesn't guarantee that event
+          will be deleted right after retention time expires.
+
+          - 'compact' cleanup policy will keep only the latest event for each event key. The compaction is performed per
+          partition, there is no compaction across partitions. The key that will be used as a compaction key should be
+          specified in 'partition_compaction_keys' field.
+          The compaction can be not applied to events that were published recently and located at the head of the queue,
+          which means that the actual amount of events recevied by consumers can be different depending on time when the
+          consumption happened.
+          When using 'compact' cleanup policy user should consider that different Nakadi endpoints showing the amount
```

Comment:
Besides adding this information here, what do you think about reviewing the text for those endpoints and make a simple reference to this attribute. So that someone reading the API starting from those endpoints also has the chance to know that the semantics change when used in combination with this feature.

Reply:
Fixed https://github.com/zalando/nakadi/pull/905/commits/ff56c178009f86fa1265685bb9a35c5e39e41897